$DragonFly: src/usr.sbin/ntpd/patches/Attic/log.c.patch,v 1.2 2005/02/28 21:34:48 joerg Exp $

--- log.c.orig	2004-11-09 20:32:47.000000000 +0100
+++ log.c	2004-11-09 20:34:22.000000000 +0100
@@ -39,12 +39,10 @@
 void
 log_init(int n_debug)
 {
-	extern char	*__progname;
-
 	debug = n_debug;
 
 	if (!debug)
-		openlog(__progname, LOG_PID | LOG_NDELAY, LOG_DAEMON);
+		openlog(getprogname(), LOG_PID | LOG_NDELAY, LOG_DAEMON);
 
 	tzset();
 }
